src/absil/il.fs

```diff
@@ -75,18 +75,16 @@ let rec splitNamespaceAux (nm:string) =
 
 /// Global State. All namespace splits ever seen
 // ++GLOBAL MUTABLE STATE
-let memoizeNamespaceTable = new Dictionary<string,string list>(10)
+let private memoizeNamespaceTable = new Dictionary<string,string list>(10)
 
 //  ++GLOBAL MUTABLE STATE
-let memoizeNamespaceRightTable = new Dictionary<string,string option * string>(100)
-
+let private memoizeNamespaceRightTable = new Dictionary<string,string option * string>(100)
```

Comment:
This guy is read from/written to further down in the file (in `splitTypeNameRight`) - synchronization should be added there, too.